src/test/java/dk/alexandra/fresco/suite/spdz/TestSpdzComparison.java

```diff
@@ -135,4 +137,27 @@ public void test_compareEQ_Sequential() throws Exception {
 		runTest(new ComparisonTests.TestCompareEQ(),
 				EvaluationStrategy.SEQUENTIAL, StorageStrategy.IN_MEMORY);
 	}
+	
+	@Test
+	public void test_isSorted() throws Exception {
+		runTest(new SortingTests.TestIsSorted(),
+				EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
+	}
+	
+	@Test
+	public void test_compareAndSwap() throws Exception {
+		runTest(new SortingTests.TestCompareAndSwap(),
+				EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
+	}
+	@Test
+	public void test_Sort() throws Exception {
```

Comment:
I would probably create a new class for testing sorting in itself. We should at some point soon (when I get the time) create a test class for all arithmetic protocol suites that fulfil the requirements. But for now, just a separate class I think. Based on: Sorting is using comparison, but it more than that.